### ISSUE TITLE
Accuracy problem

### ISSUE THREAD

**Issue by pemm-code** [linwhitehat/ET-BERT#22]
Hello,lin,At the beginning, I didn't load the pre training parameters. The accuracy rate of 10epoch can reach 87. After I loaded the pre training parameters provided by you, the accuracy rate of 10epoch is only about 82, (in the loading process, because your data is embedded to [60000, 768], and the data is only about [21100, 768], I just did not load embedded.word_embedded.weight. Other parameters are provided by you, and the results should increase a lot. Why is this case?) the entire process uses CSTNet data sets

**Comment by pemm-code** [comment]
> Hello,lin,At the beginning, I didn't load the pre training parameters. The accuracy rate of 10epoch can reach 87. After I loaded the pre training parameters provided by you, the accuracy rate of 10epoch is only about 82, (in the loading process, because your data is embedded to [60000, 768], and the data is only about [21100, 768], I just did not load embedded.word_embedded.weight. Other parameters are provided by you, and the results should increase a lot. Why is this case?) the entire process uses CSTNet data sets

Sorry to disturb you again. This problem has bothered me for four days. At present, there is no good solution, which makes the next step of work impossible to follow up. I would like to ask the author if there are some solutions or good opinions on this problem. Thank you for your work and look forward to your reply

**Comment by linwhitehat** [comment]
> > Hello,lin,At the beginning, I didn't load the pre training parameters. The accuracy rate of 10epoch can reach 87. After I loaded the pre training parameters provided by you, the accuracy rate of 10epoch is only about 82, (in the loading process, because your data is embedded to [60000, 768], and the data is only about [21100, 768], I just did not load embedded.word_embedded.weight. Other parameters are provided by you, and the results should increase a lot. Why is this case?) the entire process uses CSTNet data sets
> 
> Sorry to disturb you again. This problem has bothered me for four days. At present, there is no good solution, which makes the next step of work impossible to follow up. I would like to ask the author if there are some solutions or good opinions on this problem. Thank you for your work and look forward to your reply

Hi, thank you for your interest in our work. According to my understanding, there should be no conflict between the embedding dimension difference and the loading of the pre-trained model, where the dimensionality is corresponding to the number of samples in the fine-tuning phase. We tried to compare fine-tuning and loading of pre-trained model in CSTNET dataset, and both cases can achieve the expected experimental results. Therefore, we may need to know more about the situation to determine the reason.

![image](https://user-images.githubusercontent.com/20349381/177071315-629e6332-e20f-4868-9531-330aa7fcd8fd.png)

**Comment by pemm-code** [comment]
> > > Hello,lin,At the beginning, I didn't load the pre training parameters. The accuracy rate of 10epoch can reach 87. After I loaded the pre training parameters provided by you, the accuracy rate of 10epoch is only about 82, (in the loading process, because your data is embedded to [60000, 768], and the data is only about [21100, 768], I just did not load embedded.word_embedded.weight. Other parameters are provided by you, and the results should increase a lot. Why is this case?) the entire process uses CSTNet data sets
> > 
> > 
> > Sorry to disturb you again. This problem has bothered me for four days. At present, there is no good solution, which makes the next step of work impossible to follow up. I would like to ask the author if there are some solutions or good opinions on this problem. Thank you for your work and look forward to your reply
> 
> Hi, thank you for your interest in our work. According to my understanding, there should be no conflict between the embedding dimension difference and the loading of the pre-trained model, where the dimensionality is corresponding to the number of samples in the fine-tuning phase. We tried to compare fine-tuning and loading of pre-trained model in CSTNET dataset, and both cases can achieve the expected experimental results. Therefore, we may need to know more about the situation to determine the reason.
> 
> ![image](https://user-images.githubusercontent.com/20349381/177071315-629e6332-e20f-4868-9531-330aa7fcd8fd.png)

![3](https://user-images.githubusercontent.com/56250643/177075146-f183bde5-18eb-4a67-8f7e-4c37407fb9c6.png)
![2](https://user-images.githubusercontent.com/56250643/177075150-9f034111-e92f-4ed3-a0c9-05a228c5cfdc.png)
![1](https://user-images.githubusercontent.com/56250643/177075153-a581c3e6-c73b-442d-8e2c-0070af199bed.png)
Thank you for your reply. When I directly load your parameters (as shown in Figure 1), I report such an error (as shown in Figure 2), so I can only load them in this way, and the effect is really not ideal (as shown in Figure 3). Thank you again for your reply in your busy schedule

**Comment by pemm-code** [comment]
Sorry to bother you again, but I'm still troubled by the above problem these days and can't solve it. Does the author think the problem I encountered above is the reason for the poor effect, or does the author think the reason for the problem will appear in which link? I look forward to your reply and guidance. Good luck

**Comment by linwhitehat** [comment]
> Sorry to bother you again, but I'm still troubled by the above problem these days and can't solve it. Does the author think the problem I encountered above is the reason for the poor effect, or does the author think the reason for the problem will appear in which link? I look forward to your reply and guidance. Good luck

We may not be able to identify the problem you are experiencing at this time, and I suggest that you can re-implement the pre-training and use it for downstream task testing.
